pytential/symbolic/dof_desc.py

```diff
@@ -55,12 +58,26 @@
 
 # {{{ discretizations
 
+class UNNAMED_SOURCE:                   # noqa: N801
```

Comment:
Link to the use case in the interface for `prepare_...`. (same for `_UNNAMED_TARGET`)